Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/stitch_borders.h

```diff
@@ -33,17 +33,22 @@
 #include <CGAL/Polygon_mesh_processing/internal/named_function_params.h>
 #include <CGAL/Polygon_mesh_processing/internal/named_params_helper.h>
 #include <CGAL/Polygon_mesh_processing/connected_components.h>
+
 #include <CGAL/array.h>
+#include <CGAL/Dynamic_property_map.h>
```

Comment:
no longer needed?

Reply:
There was a 'CGAL::dynamic_face_property_t' in the file so I left it.